visualize-admin/visualization-tool
# feat: Use abbreviations

Closes #903.

This PR adds an ability to display abbreviations instead of full labels for supported dimensions (when `schema:alternateName` property is defined for dimension values).

**Supported dimensions**: all except `NumericalMeasure` and `TemporalDimension`.

**Color segmentation**: Area, Column, Line, Map, Pie, Scatterplot
**X**: Column

- [ ] **To clarify**: should the button stay visible, but disabled, or should it not be shown at all?

### How to test
1. Open [Einmalvergütung für Photovoltaikanlagen dataset](https://visualization-tool-git-feat-use-abbreviations-ixt1.vercel.app/en/browse?previous=%7B%22includeDrafts%22%3Afalse%7D&dataset=https%3A%2F%2Fenergy.ld.admin.ch%2Fsfoe%2Fbfe_ogd84_einmalverguetung_fuer_photovoltaikanlagen%2F6&dataSource=Prod).
2. Create a chart.
3. Add color segmentation (by Kanton).
4. See that an "Use abbreviations" checkbox is visible below dimension select element.
5. It should be possible to toggle abbreviations.
6. Follow the same logic for [Bathing water quality dataset](https://visualization-tool-git-feat-use-abbreviations-ixt1.vercel.app/en/browse?dataset=https%3A%2F%2Fenvironment.ld.admin.ch%2Ffoen%2Fubd0104%2F6&dataSource=Prod), but there see that the button is disabled, because no abbreviations exist for the dimensions in this dataset.
